experimental/generation/generator/packages/library/test/merge.test.ts

```diff
@@ -264,18 +264,13 @@ describe('dialog:generate --merge library', async function () {
             assertRemovedProperty(comparison, 'Toppings', errors)
             assertRemovedProperty(comparison, 'Sauces', errors)
             await assertContains('language-generation/en-us/BreadValue/sandwichMerge-BreadValue.en-us.lg', /black/, errors)
-            await assertContains('language-understanding/en-us/Bread/sandwichMerge-Bread-BreadValue.en-us.lu', /black/, errors)
             await assertMissing('language-generation/en-us/BreadValue/sandwichMerge-BreadValue.en-us.lg', /white/, errors)
```

Comment:
Maybe you want to keep these and adjust the modified schema to include the $examples with black in it.